aten/src/ATen/Utils.h

```diff
@@ -24,9 +24,9 @@ AT_API int _crash_if_asan(int);
 
 template <typename T, typename Base>
 static inline T* checked_cast_storage(Base* expr, const char * name, int pos) {
-  if (typeid(*expr) != typeid(T))
-    AT_ERROR("Expected object of type ", T::typeString(), " but found type ", expr->type().toString(),
-             " for argument #", pos, " '", name, "'");
+//  if (typeid(*expr) != typeid(T))
+//    AT_ERROR("Expected object of type ", T::typeString(), " but found type ", expr->type().toString(),
+//             " for argument #", pos, " '", name, "'");
```

Comment:
Uhhh... what's this? This can't possibly be right. I'm pretty sure you want something like https://github.com/pytorch/pytorch/pull/9764 here

Reply:
Yes, I did this to compile it and see what happens

Reply:
This is still a thing.